server/api/contest_handler.py

```diff
@@ -70,15 +70,30 @@ def get_contest(contest_id):
     except Exception:
         return jsonify("Contest does not exist")
 
+    #Load user info
     try:
         user = User.query.filter_by(id=contest.contest_creater).first()
         if user == None: 
             raise Exception
     except Exception:
         return jsonify("Contest owner not found")
 
+    #Load submissions
+    allSubmissions = Submission.query.filter_by(contest_id=contest_id).all()
+    formatedSubmissions = []
+
+    for submission in allSubmissions:
+        submiterId = submission.submiter_id
+        submiterName = User.query.filter_by(id=submiterId).first().username

```

Comment:
great work!

to improve on this, you can actually take advantage of our postgres db. If you join the tables of submissions and user (off of submiter_id), on line 82, you can reduce a lot of code here, and allow the db to do the work for you!